shell.nix

```diff
@@ -1 +1,65 @@
-(import ./default.nix {}).shell
+# This file is used by nix-shell.
+# It just takes the shell attribute from default.nix.
```

Comment:
Super super minor, I don't think this comment applies here?

Reply:
Removed line 2 from pending commit.